README.rst

```diff
@@ -82,12 +82,28 @@ By default, static assets are stored in the default Django storage backend. To o
         "STORAGE_FUNC": scorm_storage,
     }
 
-This should be added both to the LMS and the CMS settings. Instead of a function, a string that points to an importable module may be passed::
+Configuration for SCORM XBlock for AWS S3 storage
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+In order to configure the SCORM XBlock for AWS S3 storage use case, you'll need to modify the `XBLOCK_SETTINGS` in both the `lms/envs/private.py` and `cms/envs/private.py` files.
+
+Add the following lines to these files::
+
+    # XBlock settings for ScormXBlock
     XBLOCK_SETTINGS["ScormXBlock"] = {
-        "STORAGE_FUNC": "my.custom.storage.module.get_scorm_storage_function",
+        "STORAGE_FUNC": "openedxscorm.storage.s3",
+        "SCORM_S3_BUCKET_NAME": "<your-bucket-name>",
+        "SCORM_S3_QUERY_AUTH": False
```

Comment:
We don't need to prefix these settings with "SCORM_" as they are already part of the "ScormXBlock" settings. So we should rename them to "S3_BUCKET_NAME"  and "S3_QUERY_AUTH".